src/urllib3/connectionpool.py

```diff
@@ -391,6 +394,7 @@ def _make_request(
             :class:`urllib3.util.Timeout`, which gives you more fine-grained
             control over your timeouts.
         """
+
```

Comment:
nit: Remove this added newline

Reply:
I can remove, but I feel like the added white space makes the code more readable.

Reply:
done